contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -332,21 +338,18 @@ def _statsbeat_failure_reached_threshold():
     return state.get_statsbeat_initial_failure_count() >= 3
 
 
-def _update_requests_map(type, value=None):
-    if value is None:
-        value = 0
+def _update_requests_map(type_name, value=None):
+    # value is either None, duration, status_code or exc_name
     with _requests_lock:
-        if type == "count":
-            _requests_map['count'] = _requests_map.get('count', 0) + 1  # noqa: E501
-        elif type == "duration":  # value will be duration
-            _requests_map['duration'] = _requests_map.get('duration', 0) + value  # noqa: E501
-        elif type == "success":
-            _requests_map['success'] = _requests_map.get('success', 0) + 1  # noqa: E501
-        else:
-            # value will be a key (status_code/error message)
+        if value is None:  # success, count
+            _requests_map[type_name] = _requests_map.get(type_name, 0) + 1
+        elif type_name == "duration":  # value will be duration
+            _requests_map[type_name] = _requests_map.get(type_name, 0) + value  # noqa: E501
+        else:  # exception, failure, retry, throttle
```

Comment:
Duration is the only typename that will count as a success? Or at least the only such typename where we will pass a value?